src/blueapi/service/main.py

```diff
@@ -68,14 +71,44 @@ async def lifespan(app: FastAPI):
     teardown_runner()
 
 
+auth = Authentication(AuthenticationType.PKCE)
 app = FastAPI(
     docs_url="/docs",
     title="BlueAPI Control",
     lifespan=lifespan,
     version=REST_API_VERSION,
+    # https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/
+    swagger_ui_init_oauth={
+        "clientId": auth.client_id,
+        "clientSecret": auth.client_secret,
+        "usePkceWithAuthorizationCodeGrant": True,
+        "scopeSeparator": " ",
+        "scopes": "openid profile offline_access",
+    },
+)
```

Comment:
Let's leave web gui log-in out for now, we want to use the full-flow when we visit in a browser: I think the ticket was just supporting it for the CLI/Java autogenerated client.